src/api/common/helpers/jwt-auth.test.js

```diff
@@ -170,13 +285,20 @@ describe('jwt-auth', () => {
       )
 
       // Assert
-      expect(result).toBe(true)
+      expect(result).toEqual({
+        valid: true,
+        source: null,
+        sdi: undefined,
+        sbi: undefined
+      }) // tolerate sdi typo? remove
+      // Actually just:
```

Comment:
Looks like this should just be `sbi`, no?

```suggestion
        sbi: undefined
      })
```